### PR TITLE
RF-121 Show when a project is tagged as outdated

### PR DESCRIPTION
**Pull request by eronisko**
image

### CODE REVIEW

File: app/views/projects/show.html.erb [hunk 1]
```diff
@@ -64,6 +64,15 @@
     <p class="text-right mb-5 mt-1">
       Posledná aktualizácia: <strong><%= l(@project.updated_at.to_date, format: :human) %></strong></strong>
     </p>
+    <% if @project.outdated? %>
+      <div class="alert alert-warning my-2" role="alert">
+        <h5 class="alert-heading">Hodnotenie nie je aktuálne</h5>
+        <p>
```
Comment (pavolzbell) [inline app/views/projects/show.html.erb]: vizual ma ovela vacsiu medzeru na spodu ako hore, skusme `<p class="mb-0">`

File: db/schema.rb [hunk 2]
```diff
@@ -104,8 +104,10 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "title", null: false
+    t.string "tags", array: true
```
Comment (pavolzbell) [inline db/schema.rb]: v tomto pripade by bolo dobre mat `default: []`

File: app/models/project_revision.rb [hunk 3]
```diff
@@ -65,6 +65,10 @@ def load_from_data(raw)
     load_ratings(rest)
   end
 
+  def outdated?
+    tags&.include?('rf-outdated')
```
Comment (pavolzbell) [inline app/models/project_revision.rb]: bolo by vhodne mat `default: []` a neriesit `nil`

File: app/views/projects/show.html.erb (hunk 1 above)
Comment (eronisko) [reply app/views/projects/show.html.erb]: dobry postreh

**Pull request merged**